<launch>
    <arg name="map_frame" default="map" />
    <arg name="odom_frame" default="odom" />
    <arg name="base_frame" default="base_footprint" />
    <arg name="path_publish_topic_name" default="/following_path" />

    <node name="husky_panda_path" pkg="husky_panda_slam" type="husky_panda_path" output="screen">
        <param name="map_frame" value="$(arg map_frame)" />
        <param name="odom_frame" value="$(arg odom_frame)" />
        <param name="base_frame" value="$(arg base_frame)" />
        <param name="publish_topic_name" value="$(arg path_publish_topic_name)" />
    </node>
</launch>